packages/tools/src/tools/segmentation/ThresholdPreviewTool.ts

```diff
@@ -0,0 +1,235 @@
+import {
+  activeSegmentation,
+  segmentIndex as segmentIndexController,
+  config as segmentationConfig,
+  state as segmentationState,
+} from '../../stateManagement/segmentation';
+import { getSegmentation } from '../../stateManagement/segmentation/segmentationState';
+import { PublicToolProps, ToolProps } from 'tools/src/types';
```

Comment:
should not import like this, probably your autocomplete

Reply:
Done